src/components/Profile/components/OfficeInfoList/UpdateOfficeHoursDialog/index.jsx

```diff
@@ -0,0 +1,71 @@
+import { FormControl, IconButton, TextField } from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+import GordonDialogBox from 'components/GordonDialogBox';
+import GordonSnackbar from 'components/Snackbar';
+import { useState } from 'react';
+import userService from 'services/user';
+
+const UpdateOfficeHours = (props) => {
+  const [open, setOpen] = useState(false);
+  const [hours, setHours] = useState(props.officeHours);
+  const [snackbar, setSnackbar] = useState({ message: '', severity: null, open: false });
+  var maxCharacter = 4000;
+
+  const handleSubmit = async () => {
+    try {
+      await userService.updateOfficeHours(hours);
+      setSnackbar({
+        message: 'Your office hours will update within a couple hours.',
+        severity: 'success',
+        open: true,
+      });
+    } catch {
+      setSnackbar({
+        message: 'Office hours failed to update. Please contact CTS.',
+        severity: 'error',
+        open: true,
+      });
+    }
+    setOpen(false);
+  };
+
+  return (
+    <div>
+      <IconButton style={{ marginBottom: '0.5rem' }} onClick={() => setOpen(true)} size="large">
+        <EditIcon style={{ fontSize: 20 }} />
```

Comment:
```suggestion
      <IconButton style={{ marginBottom: '0.5rem' }} onClick={() => setOpen(true)} size="large">
        <EditIcon style={{ fontSize: 20 }} />
```
personal preference would be that these are stored in .css files

Reply:
same as other files in the same folder
![image](https://github.com/gordon-cs/gordon-360-ui/assets/115328139/2cb80dfa-a00c-4684-9fc6-6950257f29a9)
![image](https://github.com/gordon-cs/gordon-360-ui/assets/115328139/6984d8d9-d66f-4302-a044-181f74ee4bd6)
![image](https://github.com/gordon-cs/gordon-360-ui/assets/115328139/dbb35112-2cc5-44ad-9723-66b0e8b84475)